src/amo/utils/compatibility.js

```diff
@@ -77,6 +78,25 @@ export const isFirefox = ({
   return userAgentInfo.browser.name === 'Firefox';
 };
 
+export const isFirefoxWithOldRootCerts = ({
+  userAgentInfo,
+}: {|
+  userAgentInfo: UserAgentInfoType,
+|}): boolean => {
+  // If the userAgent is false there was likely a programming error.
+  invariant(userAgentInfo, 'userAgentInfo is required');
+
+  const majorAppVersion = parseInt(userAgentInfo.browser?.version, 10);
```

Comment:
is this working because `parseInt` will break after the first non digit?

Reply:
Yes.

Ultimately... this is what mozCompare does as well as far as I can tell: https://github.com/mozilla/addons-moz-compare/blob/3febf8e26c85eaf69fc6d7dca0a5af0df0cba6b3/src/index.js#L80

Reply:
I could use `parseVersionPart(userAgentInfo.browser?.version).a` instead or even `mozCompare(browser.version, '128') === -1` like you did but that felt a bit more obscure

Reply:
it's alright then, let's keep as is

Reply:
mmm looking again I'm starting to warm up to the idea of using `mozCompare` here, for consistency.